### PR TITLE
telco5g: Install HCP with SNO in telco jobs

### PR DESCRIPTION
**Pull request by sshnaidm**

### CODE REVIEW

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-telco5g-hcp-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-telco5g-hcp-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-telco5g-hcp-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-telco5g-hcp-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-telco5g-hcp-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-telco5g-hcp-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-telco5g-hcp-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-telco5g-hcp-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-telco5g-hcp-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-telco5g-hcp-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-telco5g-hcp-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-telco5g-hcp-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-telco5g-hcp-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-telco5g-hcp-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-telco5g-hcp-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-telco5g-hcp-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-telco5g-hcp-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-telco5g-hcp-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-telco5g-hcp-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-telco5g-hcp-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-telco5g-hcp-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-telco5g-hcp-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-telco5g-hcp-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-telco5g-hcp-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-telco5g-hcp-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-telco5g-hcp-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-telco5g-hcp-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-telco5g-hcp-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-telco5g-hcp-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-telco5g-hcp-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-telco5g-hcp-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-telco5g-hcp-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-telco5g-hcp-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-telco5g-hcp-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-telco5g-hcp-cnftests

File: ci-operator/step-registry/telco5g/hcp-cluster-setup/telco5g-hcp-cluster-setup-commands.sh [hunk 1]
```diff
@@ -125,78 +264,42 @@ cat << EOF > ~/fetch-kubeconfig.yml
...
 status=0
+
 ANSIBLE_LOG_PATH=$ARTIFACT_DIR/ansible.log ANSIBLE_STDOUT_CALLBACK=debug ansible-playbook \
-    playbooks/hosted_bm_cluster.yaml \
-    -e kubeconfig=/home/kni/${MGMT_CLUSTER}-kubeconfig \
-    -e pre_step_mgmt_cluster=false \
+    playbooks/sno_hcp_e2e.yml \
     -e hcphost=$CLUSTER_NAME \
-    -e virtual_worker_count=2 \
-    -e hide_sensitive_log=true \
```
Comment (Sandeepyadav93) [inline ci-operator/step-registry/telco5g/hcp-cluster-setup/telco5g-hcp-cluster-setup-commands.sh:193]: I think we can retain `hide_sensitive_log=true`

Comment (sshnaidm) [reply ci-operator/step-registry/telco5g/hcp-cluster-setup/telco5g-hcp-cluster-setup-commands.sh:193]: ack

**Pull request merged**